Shared/lib/Dancer/Plugin/Database/Core.pm

```diff
@@ -310,8 +310,16 @@ sub _get_connection {
 sub _check_connection {
     my $dbh = shift;
     return unless $dbh;
-    if ($dbh->{Active} && (my $result = $dbh->ping)) {
-        if (int($result)) {
+    if ($dbh->{Active}) { 
+        my $result = undef;
+
+        eval { 
```

Comment:
can you create a new version with this code fimplified?

my $result = eval { $dbh->ping; };
return 0 if $@;